steps/keras/callbacks.py

```diff
@@ -34,6 +34,23 @@ def on_epoch_end(self, epoch, logs={}):
             K.set_value(self.model.optimizer.lr, self.gamma * K.get_value(self.model.optimizer.lr))
 
 
+class UnfreezeLayers(Callback):
+    def __init__(self, unfreeze_on_epoch, from_layer=0, to_layer=1):
```

Comment:
@jakubczakon for later: we can specify `from_layer` and `to_layer` in the `pipeline_config.py` just like we specify `unfreeze_on_epoch`.